src/makeInfinityAdder.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let emptyAgain = 0;
```

Comment:
You don't need this variable, try to refactor without it. If you have any questions - feel free to ask them in `fe_chat`.